src/main/java/org/jitsi/videobridge/VideoChannel.java

```diff
@@ -472,6 +470,32 @@ public boolean expire()
                 .deRegisterRecurringRunnable(bandwidthProbing);
         }
 
+        MediaStream mediaStream = getStream();
+        if (mediaStream instanceof VideoMediaStream)
+        {
+            BandwidthEstimator.Statistics bweStats
+                = ((VideoMediaStream) mediaStream)
+                .getOrCreateBandwidthEstimator().getStatistics();
+            bweStats.update(System.currentTimeMillis());
+
+            Videobridge.Statistics videobridgeStats
+                = getContent().getConference().getVideobridge()
+                .getStatistics();
+
+            long lossLimitedSeconds = bweStats.getLossLimitedMs() / 1000;
```

Comment:
We are converting from milliseconds to seconds on a per-channel basis. Is this good enough? I expect that a lot of our calls spend very few ms (<500) in the lossDegraded state for example, and they might get cut to 0. I suggest we store these at milliseconds in Videobridge.Statistics and only convert to seconds the totals (in VideobridgeStatistics.java), or not convert to seconds at all.

Reply:
👍